odoo_backup_sh/__manifest__.py

```diff
@@ -39,5 +40,4 @@
     "auto_install": False,
     "installable": True,
     "application": True,
-    "uninstall_hook": "uninstall_hook",
```

Comment:
Why it's deleted?

Reply:
There is already uninstall_hook key. Line 38

Reply:
Then you need to remove the one with None value

Reply:
Done